pytest_deep_analysis/checkers.py

```
@@ -82,6 +82,30 @@ def __init__(self, linter: "PyLinter"):
+        """
+        import json
+        from pathlib import Path
+
+        cache_file = Path(".pytest_deep_analysis_cache.json")
+        if not cache_file.exists():
+            return {}
+
+        try:
+            cache_data = json.loads(cache_file.read_text())
+            return cache_data.get("tests_with_semantic_validation", {})
+        except Exception:
```

Ruff (0.14.4)

Do not catch blind exception: `Exception`

(BLE001)